### PR TITLE
Add initial skeleton and code for NW salt code

### PR DESCRIPTION
**Pull request by arbulu89**
Firs implementation of the Netweaver salt module.

The scope of this PR is to have the minimum modules and methods to install the ASCS SAP instance

### CODE REVIEW

File: salt/modules/netweavermod.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    '''
+    Check if SAP Netweaver platform is installed
+
+    sid
+        Netweaver system id (PRD for example)
+    inst
+        Netweaver instance number (00 for example)
+    password
+        Netweaver instance password
+    sap_intance
```
Comment (diegoakechi) [inline salt/modules/netweavermod.py]:
```suggestion
    sap_instance
```

File: salt/states/netweavermod.py [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+
+def _get_sap_instance_type(product_id):
+    '''
+    Get SAP instance type from the product id string
+    '''
+    return product_id.split(':')[0].split('-')[-1].lower()
+
+
+def installed(
```
Comment (diegoakechi) [inline salt/states/netweavermod.py]: @arbulu89 Wouldn't you change this to `present`?

Comment (arbulu89) [reply salt/states/netweavermod.py]: I would like... But i don't want to have `present` here and `installed` in HANA.
I think we will need to do the change in all of them together.

Anyway, what do you prefer @diegoakechi @Simranpal ? `installed/uninstalled` vs `present/absent`.
The 2nd follows more the salt style in my opinion

Comment (diegoakechi) [reply salt/states/netweavermod.py]: I prefer to follow the salt convention, but I am ok to do it in a special PR changing all at the same time.

Comment (Simranpal) [reply salt/states/netweavermod.py]: I think installed/uninstalled is consistent with HANA

**Pull request merged**